vrk-kpa/suomidigi
# SUOPA-772 Breadcrumb

- To test: 
- `make fresh drush-cr`
- Explore the site in desktop and mobile and check that the breadcrumbs work and look okay
   - **Note**. The front page should not have breadcrumb.
